.github/workflows/deploy.yml

```diff
@@ -62,6 +62,32 @@ jobs:
         terraform workspace select staging terraform/app || terraform workspace new staging terraform/app
         terraform apply -var-file terraform/workspace-variables/staging.tfvars -auto-approve -input=false terraform/app
 
+    - name: Set up Ruby
```

Comment:
This repeats what's in the smoke test workflow. Could we try to reuse it instead?
Maybe by using a combination like:
- [Workflow dispatch](https://github.com/marketplace/actions/workflow-dispatch) to trigger the smoke test workflow from the deploy workflow
- [Wait on check](https://github.com/marketplace/actions/wait-on-check) to wait in deploy workflow for smoke test workflow to end